back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -65,12 +61,12 @@ public ResponseEntity<String> makeRequest(/*@DTO(CertificateRequestCreateDto.cla
 	public ResponseEntity<String> acceptRequest(@PathVariable Long id) {
 		CertificateRequest req = certificateRequestService.findByIdAndStatus(id, Status.PENDING);
 		
-		User issuee = userService.findById(1L); // TODO: Fetch user ID from token.
-		if (issuee == null) {
+		User issuer = userService.findById(1L); // TODO: Fetch user ID from token.
+		if (issuer == null) {
 			return new ResponseEntity<>("User does not exist", HttpStatus.NOT_FOUND);
 		}
 		
-		List<CertificateRequest> requests = certificateRequestService.findByIssuee(issuee);
+		List<CertificateRequest> requests = certificateRequestService.findByIssuee(issuer);
 		if (!requests.contains(req)) {
 			// return new ResponseEntity<>("Request not found", HttpStatus.NOT_FOUND);
 		}
```

Comment:
What does this even do? Gets a list, and then does nothing?

Reply:
Altered in [e0e6910](https://github.com/magley/sssscs/pull/5/commits/e0e691099ce41198adfd10b1fb405b75e63cf07b). It has to be uncommented until we actually fetch the real user making the http request, because the program will always return requests issued to user with id=1.